hc/accounts/tests/test_check_token.py

```diff
@@ -22,7 +22,13 @@ def test_it_redirects(self):
         self.assertEqual(self.profile.token, "")
 
     ### Login and test it redirects already logged in
+    def test_login_redirects_if_already_loggedin(self):
+        r = self.client.post("/accounts/check_token/alice/secret-token/")
+        self.assertRedirects(r,"/checks/")
```

Comment:
maybe assert that a user who is not logged in is succesfully logged in then try to log in the exact same user and assert that they are redirected to `checks`